cylc/uiserver/resolvers.py

```diff
@@ -346,22 +319,16 @@ async def play(cls, workflows, args, workflows_mgr, log):
                         f'Could not start {tokens["workflow"]}'
                         f' - {cmd_repr}'
                     )
-                    raise Exception(
-                        msg
-                    )
+                    raise Exception(msg)
 
             except Exception as exc:
                 # oh noes, something went wrong, send back confirmation
                 return cls._error(exc)
 
-            else:
-                # send a success message
-                return cls._return(
-                    'Workflow started'
-                )
```

Comment:
Looks like the original intention was to return a list with the status of each workflow which makes sense.

https://github.com/cylc/cylc-uiserver/commit/2f3a80a8eaafe4cb981332d8894c51391e901b5a#diff-6cea667f14824a3338879a9b66beae2f50139ec3d0ba7e0670110ba9debf13d7R173-R190

Reply:
The UI only accepts a response of the format `[bool, message]` where `bool` indicates the success/failure or just `message` which implies success

Reply:
So something like this:

```
[
    False,
    [
        'abc: started',
        'bcd: started',
        'cde: error, workflow was previously running with Cylc 8.0.1, use --upgrade to restart',
    ]
]
```

Reply:
Done